docs/administration-guide/13-embedding.md

```diff
@@ -126,20 +139,40 @@ To change the look and feel of the embed, you can add additional parameters to t
 
 - **bordered**: true or false. Adds a visible border to the embed.
 - **titled**: true or false. Adds or removes the title to the embedding.
-- **theme**: null or night. Shows the embedded iframe in dark mode.
+- **theme**: light, transparent, or night. Shows the embedded iframe in in light (theme is null), transparent (no background, just the ), or dark mode.
 
-For example: 
+For example:
 
 ```
 http://yourmetabaseurl.com/embed/dashboard/a_very_huge_hashed_url#theme=night&hide_parameters=category&titled=true&bordered=false
 ```
 
 These settings can also be changed under **Style** when previewing the embedded question or dashboard and its code in Metabase.
 
-## Reference applications
+## Advanced embedding features
+
+{% plans-blockquote feature="Advanced embedding features" %}
+
+Paid plans give you additional customization options for embedded items.
+
+### Remove the Powered by Metabase banner
```

Comment:
```suggestion
### Remove the "Powered by Metabase" banner
```

Style question... do we want to use quotation marks in a situation like this, where the text is usually bolded, but it's used in a heading where that emphasis is lost?